### PR TITLE
Fixed #26098 -- Changed OpenLayers.js references to cdnjs.

### PR DESCRIPTION
**Pull request by blueyed**
https://code.djangoproject.com/ticket/26098

### CODE REVIEW

**Comment by timgraham** [comment]
Could you please add 1.11 release notes as well as `versionchanged` annotations as described in the [Patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist).

File: docs/ref/contrib/gis/forms-api.txt [hunk 1]
```diff
@@ -153,16 +153,14 @@ Widget classes
...
-    can host a copy of ``openlayers.js``
-    `tailored to your needs`_ on your own server or refer to a copy from a
-    content-delivery network like https://cdnjs.com/. This will also allow
-    you to serve the JavaScript file(s) using the ``https`` protocol if needed.
+    hosted on the ``cdnjs.cloudflare.com`` content-delivery network.
+    You can subclass these widgets in order to specify your own version of the
+    ``openlayers.js`` file `tailored to your needs`_ in the ``js`` property of
+    the inner ``Media`` class (see :ref:`assets-as-a-static-definition`).
+
+    When using :class:`OSMGeoAdmin` (derived from :class:`GeoModelAdmin`), you
```
Comment (timgraham) [inline docs/ref/contrib/gis/forms-api.txt]: You need to use the full path in the `:class:` annotations to fix the broken links that are causing the docs build failure.

**Comment by blueyed** [comment]
Ok for the release notes, but do you want the `versionchanged` for the changed JS URL really?

**Comment by timgraham** [comment]
Yes, why wouldn't it be appropriate?

**Comment by blueyed** [comment]
It's not a new feature, is it?  And it wasn't done for the update in 41ed633.
What should the note say?  That it is now hosted on a CDN and through https?

**Comment by timgraham** [comment]
@claudep, any concerns here after the docs are fixed up?

**Comment by claudep** [comment]
I'm fine with that patch, thanks!

**Comment by timgraham** [comment]
Updated and merged in b1d6b0a7b121eec860b518b4903d7c8c74f7773b, thanks!

**Comment by merwok** [comment]
Hello!  Any chance of a 1.10 backport?

**Comment by timgraham** [comment]
Per our [supported versions policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions), 1.10 is only receiving fixes for security and data loss issues.

**Comment by merwok** [comment]
Another reason to upgrade!  Thanks for the answer, I’ll bookmark the link.

**Pull request closed without merging**